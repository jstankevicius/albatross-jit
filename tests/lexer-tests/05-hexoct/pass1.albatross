0x0
0xCAFEBA 0xDEAD 0xBEEF 0xABC0DEF 0x12034 0x5607890
0x00000000ABCDEF0 0x0000000000000023456789
0x00CAFEBA 0x0DEAD 0xBEEF 0xABC0DEF 0x12034 0x5607890
