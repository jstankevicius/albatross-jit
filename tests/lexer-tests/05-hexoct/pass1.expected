1 1 INT 0
1 2 INT 13303482
10 2 INT 57005
17 2 INT 48879
24 2 INT 180096495
34 2 INT 73780
42 2 INT 90208400
1 3 INT 180150000
19 3 INT 591751049
1 4 INT 13303482
12 4 INT 57005
20 4 INT 48879
27 4 INT 180096495
37 4 INT 73780
45 4 INT 90208400
